internal/testplanet/satellite.go

```diff
@@ -116,8 +126,8 @@ func (planet *Planet) newSatellites(count int) ([]*SatelliteSystem, error) {
 				RefreshConcurrency: 2,
 			},
 			Metainfo: metainfo.Config{
-				DatabaseURL:          "bolt://" + filepath.Join(storageDir, "pointers.db"),
-				MinRemoteSegmentSize: 0, // TODO: fix tests to work with 1024
+				DatabaseURL:          "", // not used
```

Comment:
should we remove this or better to leave here as empty string?

Reply:
I try to keep all the config arguments visible to make it clear what they are set to.